torch/_dynamo/config.py

```diff
@@ -60,6 +60,10 @@
 # don't specialize on shapes and strides and put shape ops in graph
 dynamic_shapes = os.environ.get("TORCHDYNAMO_DYNAMIC_SHAPES") == "1"
 
+# This flag temporarily changes the behavior of dynamic_shapes=True.
```

Comment:
This is written weird - it doesn't "temporarily change", rather, it's a temporary flag...